### PR TITLE
chore: run prettier on entire codebase

### PR DESCRIPTION
**Pull request by blumamir**
seems like husky can very easily stop working in monorepos.
this PR is to align our entire codebase to the prettier rules.

### CODE REVIEW

File: packages/instrumentation-sequelize/test/sequelize.spec.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ import { ContextManager } from '@opentelemetry/context-base';
 import { DatabaseAttribute, GeneralAttribute } from '@opentelemetry/semantic-conventions';
 import expect from 'expect';
 
+// should be available in node_modules from sequelize installation
+const Promise = require('bluebird');
```
Comment (nirsky) [inline packages/instrumentation-sequelize/test/sequelize.spec.ts:12]: why is this here? 😮

**Review by nirsky — APPROVED** [review]

**Comment by blumamir** [comment]
fixes #83
